aws/resource_aws_launch_template.go

```diff
@@ -308,6 +308,20 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 				Optional: true,
 			},
 
+			"license_specification": {
+				Type:     schema.TypeList,
```

Comment:
Two questions:
* Does the API currently allow more than 1 specification? If not, we can set `MaxItems: 1` and the next question is moot
* Is the ordering from the API guaranteed/matter? If not, we should use `Type: schema.TypeSet`